src/common/runtime/src/global.rs

```diff
@@ -19,13 +19,36 @@ use std::sync::{Mutex, Once};
 use common_telemetry::info;
 use once_cell::sync::Lazy;
 use paste::paste;
+use serde::{Deserialize, Serialize};
 
 use crate::{Builder, JoinHandle, Runtime};
 
 const READ_WORKERS: usize = 8;
 const WRITE_WORKERS: usize = 8;
 const BG_WORKERS: usize = 8;
 
+/// The options for the global runtimes.
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
+pub struct RuntimeOptions {
+    /// The number of threads to execute the runtime for global read operations.
+    pub read_rt_size: usize,
+    /// The number of threads to execute the runtime for global write operations.
+    pub write_rt_size: usize,
+    /// The number of threads to execute the runtime for global background operations.
+    pub bg_rt_size: usize,
+}
+
+impl Default for RuntimeOptions {
+    fn default() -> Self {
+        let two_fold_cpus = num_cpus::get() * 2;
+        Self {
+            read_rt_size: two_fold_cpus,
+            write_rt_size: two_fold_cpus,
+            bg_rt_size: two_fold_cpus,
+        }
```

Comment:
Using 2x num_cpus as runtime size may be too large. I prefer to use num_cpus or a smaller size.

Reply:
I think 2x cpus are better, each core can deal with more work, improve general throughput, just like intel's hyper-threading. But I'm OK with setting them to cpu num. Let's hear other people say.

Reply:
`num_cpus` already counts logical core num. Note that we have 3 runtimes so we already use 3x cpu num.